polylint.js

```diff
@@ -28,16 +28,20 @@ var polylint = function polylint(path, options) {
   options.filter = function(){
     return false;
   };
-  options.redirect = "bower_components";
+  if (!('redirect' in options)) {
+    options.redirect = "bower_components";
+  }
   return hydrolysis.Analyzer.analyze(path, options).then(function(analyzer){
-    var lintErrors = [];
-    for (var linter in linters) {
-      lintErrors = lintErrors.concat(linters[linter](analyzer, path));
```

Comment:
This is a call as a method, whereas the below calls the linter as a function with a *thisValue* of `null`.